lib/evss/gi_bill_status/gi_bill_status_response.rb

```diff
@@ -5,19 +5,11 @@
 module EVSS
   module GiBillStatus
     class GiBillStatusResponse < EVSS::Response
-      attribute :first_name, String
-      attribute :last_name, String
-      attribute :name_suffix, String
-      attribute :date_of_birth, String
-      attribute :va_file_number, String
-      attribute :regional_processing_office, String
-      attribute :eligibility_date, String
-      attribute :delimiting_date, String
-      attribute :percentage_benefit, Integer
-      attribute :original_entitlement, Integer
-      attribute :used_entitlement, Integer
-      attribute :remaining_entitlement, Integer
-      attribute :enrollments, Array[Object]
+      attribute :body, EVSS::GiBillStatus::Post911GIBillStatus
```

Comment:
I'm don't like that there's an entire object with one attribute called `body`.  I think the `FaradayResponse.body` is just fine and that this abstraction is redundant.